### PR TITLE
Link getting started in readme to installation instructions

### PR DESCRIPTION
**Pull request by bfirsh**

### CODE REVIEW

**Comment by thaJeztah** [comment]
Should we point users to the Linux, Mac and Windows "getting started" guides? https://docs.docker.com/linux/started/, https://docs.docker.com/mac/started/, https://docs.docker.com/windows/started/

Or the install guides, which also feature Docker Toolbox; https://docs.docker.com/engine/installation/mac/, https://docs.docker.com/engine/installation/windows/, https://docs.docker.com/engine/installation/

There's a lot of overlap between those (getting started / installation guide)

@moxiegirl which one do you think is more appropriate?

File: README.md [hunk 1]
```diff
@@ -143,15 +143,16 @@ as they can be built by running a Unix command in a container.
...
 ===============
 
-Docker can be installed on your local machine as well as servers - both
-bare metal and virtualized.  It is available as a binary on most modern
-Linux systems, or as a VM on Windows, Mac and other systems.
+The easiest way to get started with Docker in development is to use [Docker
+Toolbox](https://www.docker.com/docker-toolbox). It is an installer for Mac OS X
+and Windows that installs everything you need to build apps with Docker.
+
+To install on Linux or on servers, check out the [full installation instructions
```
Comment (moxiegirl) [inline README.md]: Toolbox is only intended for Mac and Windows, so I think this is misleading for users.  The audience for the README on GitHub is expected to be more technical not less. So, I think we should point them here:

http://docs.docker.com/engine/installation/

**Comment by bfirsh** [comment]
@thaJeztah @moxiegirl Updated with your comments. Thanks!

File: README.md [hunk 2]
```diff
@@ -143,15 +143,13 @@ as they can be built by running a Unix command in a container.
 Getting started
 ===============
 
-Docker can be installed on your local machine as well as servers - both
-bare metal and virtualized.  It is available as a binary on most modern
-Linux systems, or as a VM on Windows, Mac and other systems.
+Docker can be installed either on your computer for building applications or
+on servers for running them. To get started, [check out the installation
+instructions in the documentation](http://docs.docker.com/engine/installation/).
```
Comment (thaJeztah) [inline README.md]: can you change this to `https://` ?

**Comment by thaJeztah** [comment]
One nit, but LGTM when fixed

**Comment by bfirsh** [comment]
fixed!

**Comment by thaJeztah** [comment]
Thanks!

ping @moxiegirl for a final look :)

**Pull request merged**